sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -556,6 +556,98 @@ Parameters:
 
 |===
 
+==== Dynamic Command Groups
+
+[source,c++]
+----
+namespace ext::oneapi::experimental{
+class dynamic_command_group {
+public:
+  dynamic_command_group(
+      command_graph<graph_state::modifiable> graph, std::vector<std::function<void(handler &)>> cgfList);
+
+  void set_active_cgf(int cgfIndex);
```

Comment:
For this index based approach I'm not sure if an  `int get_active_cfg()` would also be useful, also should the index parameter be unsigned?

Reply:
I think creating a `get_active_cgf()` might make sense.
I think I got too inspired by `set_arg()` using an int for index. Will change this parameter to `size_t`